recipes/trackplot/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "trackplot" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: trackplot
+  version: {{ version }}
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3e426c06e02de21817855a1e62c567410bf35c94514f72d326d686f93ba508a8
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+
+requirements:
+  host:
+    - python
+    - wheel
+    - setuptools
+    - setuptools_scm
+    - pip
+    - filetype
+    - pysam
+    - pybigwig
+    - hicmatrix
+    - click-option-group
+    - poetry
```

Comment:
try this:

```suggestion
requirements:
  host:
    - python
    - poetry
```

Reply:
Hi daler,
 Thanks for your suggests, meta.yaml changed as your suggestions
Best